kubernetes_deploy/live-1/production/deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     type: RollingUpdate
     rollingUpdate:
       maxUnavailable: 0
-      maxSurge: 1
+      maxSurge: 100%
```

Comment:
Why are you making this change? Why not let the rolling update deploy it one by one. This means during the deployment there will be a big spike in the resource consumption. 

Are you trying to prevent the application from running 2 different versions during the deployment?

Reply:
I just read ur PR description. Approving it.